OrbitLinuxTracing/TracerThread.h

```diff
@@ -5,8 +5,8 @@
 #ifndef ORBIT_LINUX_TRACING_TRACER_THREAD_H_
 #define ORBIT_LINUX_TRACING_TRACER_THREAD_H_
 
+#include <Function.h>
```

Comment:
I would say use <> for standard C/C++ includes (libc/man/linux/libc++) and "" for everything else.

Reply:
Done.